src/biokbase/narrative/tests/test_job.py

```diff
@@ -147,33 +154,39 @@ def state_2_kwargs(state, transform_list=JOB_ATTRS):
     return kwargs
 
 
-def create_job_from_ee2(job_id):
+def create_job_from_ee2(job_id, mode, children=None):
     """
     create a job object from raw job data
     """
     state = get_test_job(job_id)
-    kwargs = state_2_kwargs(state)
 
-    return Job.from_attributes(**kwargs, ee2_state=get_test_job(job_id))
+    if mode == "attributes":
+        kwargs = state_2_kwargs(state)
+        job = Job.from_attributes(**kwargs, children=children)
+    elif mode == "state":
+        job = Job.from_state(state, children=children)
+
+    return job
 
 
-def create_state_from_ee2(job_id, with_job_input=False, with_job_output=False):
+def create_state_from_ee2(job_id, mode):
     """
     create the output of job.state() from raw job data
     """
     state = get_test_job(job_id)
-    omitted_fields = EXCLUDED_JOB_STATE_FIELDS.copy()
-
-    if not with_job_output:
-        omitted_fields.append("job_output")
 
-    if with_job_input:
-        omitted_fields.remove("job_input")
```

Comment:
These seemed to be used congruently with the job status and so seemed harmless to remove